### ISSUE TITLE
Add start button functionality

### ISSUE THREAD

**Issue by camchardukian** [camchardukian/fcc-react-typescript-pomodoro-clock#7]
Add functionality to the start button, ideally such that the following two unit tests from FreeCodeCamp pass:

```
User Story #18: When I first click the element with id="start_stop", the timer should begin running from the value currently displayed in id="session-length", even if the value has been incremented or decremented from the original value of 25.

User Story #19: If the timer is running, the element with the id of time-left should display the remaining time in mm:ss format (decrementing by a value of 1 and updating the display every 1000ms).
```

**Comment by camchardukian** [comment]
The start button's basic functionality was implemented in PR #8.

**Comment by camchardukian** [comment]
All unit tests are passing now after having merged PR #20.
